packages/api/test/analytics-test.js

```diff
@@ -64,7 +64,7 @@ describe('Flaky-Analytics', () => {
     for (let i = 0; i < testCases.length; i++) {
       const testCaseObjs = [];
       for (let k = 0; k < testCases[i].length; k++) {
-        testCaseObjs.push(new TestCaseRun(testCases[i][k] ? 'ok' : 'not ok', k, k.toString()));
+        testCaseObjs.push(new TestCaseRun(testCases[i][k] ? true : false, k, k.toString()));
```

Comment:
Too many args in this one, too, sorry. Should be: `testCaseObjs.push(new TestCaseRun(testCases[i][k], k.toString()));`